tests/Library/Core/RequestTest.php

```diff
@@ -473,4 +473,17 @@ public function testUrlEquivalence() {
 
         $this->assertSame($req->getUrl(), $req->url('', true));
     }
+
+    /**
+     * Test basic attribute accessors.
+     */
+    public function testAttributeAccessors(): void {
+        $r = new Gdn_Request();
+
+        $this->assertNotFalse($r->getAttribute('foo'));
```

Comment:
Since the default of `getAttribute` is `null`, would this be a better assertion to make? 

```suggestion
        $this->assertNull($r->getAttribute('foo'));
```

Reply:
Autocomplete fail right there.